src/app/components/dropdown/dropdown.component.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../../styles/variables.scss";
+
+mat-form-field {
+    display: flex;
+    text-align: center;
+    color: $color-primary-500;
+    font-size: $font-button;
+    font-weight: $font-semibold;
+    ::ng-deep .mat-select-placeholder {
```

Comment:
Avoid using  ::ng-deep since it's deprecated. Consider adding an HTML class and then style it. If the style from your custom class won't be rendered in the browser move the declaration into styles.scss